_posts/2023-11-09-1.1.2-release.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title:  "Release 1.1.2"
+date:   2023-11-10 00:00:00 +0200
+tags: release
+author: lego11
+---
+
+Yet another update has been beamed to your computers now!
+This time starring:
+
+# New features
+
+## Automation improvements
+
+It is now possible to very easily build an automated station for APT or LRPT satellites, advantageously replacing previous setups that required anywhere from 5 to 12 separate pieces of software.
+
+lego11 has made [a detailed configuration guide](https://www.a-centauri.com/articoli/automating-satellite-reception-with-satdump) for such a station.
```

Comment:
do we move the guide on docs.satdump.org already or not?

Reply:
I'd say yes